### DISCUSSION TITLE
Adobe Animate features that need to be in Toonsquid

### DISCUSSION THREAD

**Discussion by ArtsyStar1** [keiwando/toonsquid#413] [Ideas]
Since Adobe will probably never create an iPad app version of Animate, here’s a few features from Animate itself that could be added in the future updates of Toonsquid:  
1. Bitmap fill: for someone like me who works with textures, that feature will be very useful. 
2. PSD import: It’s a very necessary feature imo.
3. Break Apart: It helps break apart symbols. 
4. Trace Image: Rather self explanatory. 
5. SWF/Flash import: I have something to confess, I think animating directly on Adobe Animate all the time is a pain in the butt simply because it’s an app for desktop/computers(I’m more of an iPad person) and I would like to import my Adobe Animate projects into Toonsquid one day. 
6. Tweening: Mainly Classic Tweening but Motion Tweening can be added too, I guess.

**Comment by trinhpxl** [comment]
confused abt 6, we already have tweening in toonsquid ?

**Reply by ArtsyStar1** [reply]
Sorry, I forgot that feature existed.

**Comment by VeiloPot** [comment]
We do have classic tweening, and for motion tweening i think that is different and something we dont have yet. Like this for example: ![image](https://github.com/keiwando/toonsquid/assets/135170331/8ad90a2d-a02b-44a0-a04c-6cae58d24574)

**Reply by ArtsyStar1** [reply]
Oh! Thanks for telling me, I forgot that feature existed.

**Comment by overcesium** [comment]
I do wish to have a break apart symbol feature, my animate process for frame by frame used to be a constant group/ ungroup (to work on a precise selection isolated from the rest of the drawing) and symbol/ unsymbol
